src/commands/transfer.ts

```diff
@@ -31,7 +32,19 @@ export async function transferCommand(
 
     const { currentWallet, wallets } = walletsData;
 
-    const wallet = wallets[currentWallet];
+    let wallet = wallets[currentWallet];
+
+    if (name) {
+      if (!wallets[name]) {
+        console.log(
+          chalk.yellow(
+            "⚠️ Wallet with the provided name does not exist. Using current wallet."
```

Comment:
If the user did a typo, this assumption can transfer funds from an undesire wallet.   Best path here is to fail the tx and let the user review the command and try again